tests/util.py

```diff
@@ -119,26 +159,99 @@ def assertSequenceAlmostEqual(
             )
         self.assertEqual(len(seq1), len(seq2))
         for a, b in zip(seq1, seq2):
-            self._assertAnyAlmostEqual(a, b, places, ignore_keys)
+            self._assertAnyAlmostEqual(
+                a,
+                b,
+                places,
+                unordered_lists=unordered_lists,
+                ignore_keys=ignore_keys,
+            )
+
+    def assertSequenceAlmostEqualIgnoreOrder(
+        self,
+        seq1: Sequence,
+        seq2: Sequence,
+        cmp: Callable,
```

Comment:
```suggestion
        cmp: Callable | None,
```

comparator can be None

Reply:
\+